fixtures/ssr2/server/render.js

```diff
@@ -36,14 +38,16 @@ module.exports = function render(url, res) {
     console.error('Fatal', error);
   });
   let didError = false;
-  const data = createServerData();
   const {pipe, abort} = renderToPipeableStream(
     <DataProvider data={data}>
       <App assets={assets} />
     </DataProvider>,
     {
       bootstrapScripts: [assets['main.js']],
-      onCompleteShell() {
+      onAllReady(){
```

Comment:
this one is not required, right?

Reply:
Nope, just added it for the ease of understanding all the various methods exposed and available. I could remove it, if required 👍